src/scss/index.scss

```diff
@@ -63,6 +64,19 @@ $header-border-color              : $red !default;
             z-index: zIndex(high);
         }
     }
+
+    // Styles for a sticky header on mobile
+    @include media('<mid') {
+        &.is-sticky {
+            left: 0;
+            top: -60px;
+            position: fixed;
```

Comment:
Late to the game – just to check – doesn't `is-sticky` helper already add this line?